### ISSUE TITLE
Type conversion issues when retrieving job logs

### ISSUE THREAD

**Issue by mannau** [rstudio/connectapi#193]
When retrieving job logs from rsconnect I encountered various errors when converting types - mainly:

1. Converting to list() type which is actually a character (of length 1)
```
Error in `ensure_column()`:
! Can't convert `col` <character> to <list>.
```
2. Converting a zero-length logical(0) to POSIXct
```
Error in `coerce_datetime()`:
! Can't convert `end_time` <logical> to match type of `to` <datetime<UTC>>.
```
 
I use the following code to retrieve logs:

```r
item <- connectapi::content_item(client, guid[i])
jobs <- connectapi::get_jobs(item)

logs <- dplyr::bind_rows(lapply(jobs$key, function(k) {
      connectapi::get_job(item, k) # -> Throws Error
}))
```

The pull request #192 Fixes the mentioned issues.

**Comment by mannau** [comment]
Any updates on this?

**Comment by toph-allen** [comment]
Closing as this was fixed by #239 and released in 0.2.0.
